_layouts/glossary-ltr.html

```diff
@@ -0,0 +1,40 @@
+---
+---
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+<head>
+    <title>{{site.title}}</title>
+    <link href="{{ '/static/site.css' | relative_url }}" rel="stylesheet" type="text/css" />
+    <link rel="stylesheet" type="text/css" href="{{site.baseurl}}/assets/css/glossary.css" />
+    <link rel="stylesheet" type="text/css" href="{{ relative_root_path }}/assets/css/glossary.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+</head>
+<body style="width: 100%">
+<main>
+    {% include title.html %}
```

Comment:
I think that's a good idea, but it's not something I quite know how to do very well, yet. I originally started with the two new layout files inheriting from the page layout, but that was causing some issues.

The right-to-left text caused further issues in styling the page, because the text was very stubbornly set against the right margin, particularly the heading. That's why that file doesn't even use title.html at this point.

I'm happy to try other implementations if people know of better ways. I like learning new things.

Reply:
let's merge as it is now and we can improve this in the future.